### PR TITLE
fix / 104: Improve naming

### PR DESCRIPTION
**Pull request by ruteranders**

### CODE REVIEW

File: openapi/event-reporting/api-spec.json [hunk 1]
```diff
@@ -353,13 +353,13 @@
             "example": "2023-01-01T13:00:00Z",
             "default": "null"
           },
-          "leftStopPlaceRef": {
+          "leftCancelledStopPlaceRef": {
             "type": "string",
             "description": "Required if `category` is `CANCELLATION` and `subcategory` is `PARTIAL`, ignored\notherwise. An NSR reference to one of the 2 stop places delimiting the sequence of\nstops affected by the partial cancellation, or to the single stop if only one stop is\naffected. See `rightStopPlaceRef`.\n",
```
Comment (eivindga) [inline openapi/event-reporting/api-spec.json]: Please update the description for this field to reference the new name: 

`rightCancelledStopPlaceRef`

File: openapi/event-reporting/api-spec.json [hunk 2]
```diff
@@ -353,13 +353,13 @@
...
             "example": "NSR:StopPlace:123",
             "default": "null"
           },
-          "rightStopPlaceRef": {
+          "rightCancelledStopPlaceRef": {
             "type": "string",
             "description": "Required if `category` is `CANCELLATION` and `subcategory` is `PARTIAL`, ignored\notherwise. An NSR reference to the other of the 2 stop places delimiting the sequence\nof stops affected by the partial cancellation, or to the single stop if only one stop\nis affected. See `leftStopPlaceRef`.\n",
```
Comment (eivindga) [inline openapi/event-reporting/api-spec.json]: Please update the description for this field to reference the new name:

`leftCancelledStopPlaceRef`

File: openapi/event-reporting/api-spec.json (hunk 1 above)
Comment (ruteranders) [reply openapi/event-reporting/api-spec.json]: Fixed!

File: openapi/event-reporting/api-spec.json (hunk 2 above)
Comment (ruteranders) [reply openapi/event-reporting/api-spec.json]: Fixed!

**Review by eivindga — APPROVED** [review]

**Pull request merged**